packages/@aws-cdk/aws-codepipeline-actions/test/cloudformation/pipeline-actions.test.ts

```diff
@@ -1,4 +1,5 @@
 import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { NotifyOnEventOptions, Rule } from '@aws-cdk/aws-codestarnotifications';
```

Comment:
Please use a qualified import instead:

```suggestion
import * as notifications from '@aws-cdk/aws-codestarnotifications';
```

Reply:
Already adjusted.